src/appleseed/renderer/modeling/aov/aov.h

```diff
@@ -92,6 +92,9 @@ class APPLESEED_DLLSYMBOL AOV
     // Apply any post-processing needed to the AOV image.
     virtual void post_process_image(const Frame& frame);
 
+    // Write image to EXR file.
```

Comment:
*OpenEXR*

Reply:
You called the method `write_images()` (plural), is this intended?

Reply:
I think standalone method in `frame.cpp` was called plural, and I made copy of that.

Reply:
Ok.